src/views/PatronsPreRegistrationListContainer/hooks/useProcessPreRegisteredUser.js

```diff
@@ -0,0 +1,92 @@
+import { useMutation } from 'react-query';
+import { useHistory } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+
+import { useOkapiKy, useCallout } from '@folio/stripes/core';
+
+import { PATRON_PREREGISTRATIONS_API, PERMISSIONS_API } from '../../../constants';
+import { navigateToUserView } from '../../../utils/utils';
+
+const getFullName = (user) => {
```

Comment:
https://github.com/folio-org/stripes-util/blob/master/lib/getFullName.js

Reply:
This stripes-util will not apply here as the pre-registration record structure is different from the user record structure

Reply:
Oh, I see

Reply:
I don't know why, but github removes my comment :D

please make an adapter for this util

Reply:
what do mean by adapter?

Reply:
by adapter I mean adapter, what about GoF patterns?